src/asm/rgbasm.5

```diff
@@ -259,12 +278,11 @@ start of an array to 256 bytes to optimize the code that accesses it.
 .Sy Hint :
 If you think this is a lot of typing for doing a simple
 .Dq org
-type thing you can quite easily write an intelligent macro (called
-.Ic ORG
-for example) that uses
+type thing you can quite easily write an intelligent macro
+.Pq called Ic ORG No for example
```

Comment:
Although there’s nothing wrong with using `Pq` here, in general there’s no reason to prefer it over () except for the situation where the first word of what you’re parenthesizing is marked up.

    .\" This is wrong, because it renders with space before ORG
    (
    .Ic ORG )

    .\" This displays correctly
    .Pq Ic ORG

Nothing wrong with `No` either but I also try to avoid it to reduce the risk of confusion. So even if I used a parenthesis macro here, I’d do this:

    .Po
    called
    .Ic ORG
    for example
    .Pc

Reply:
Reverting...